docs/admin/federation/index.md

```diff
@@ -0,0 +1,111 @@
+---
+---
+This guide explains how to set up a federation cluster. Federation is an alpha feature which lets you control multiple kubernetes clusters.
+[Federation proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/federation.md) has more details.
+
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running kubernetes cluster.
```

Comment:
Kubernetes is capitalized. This is true throughout the PR, I'm only mentioning it once here.